app.vue

```diff
@@ -1,56 +1,104 @@
 <template>
-  <div class="mb-20">
-    <GMapMap
-        :center="center"
-        :zoom="15"
-        :options="{
-            zoomControl: true,
-            mapTypeControl: false,
-            scaleControl: false,
-            streetViewControl: false,
-            rotateControl: false,
-            fullscreenControl: true,
-          }"
-        style="width: 100vw; height: 100vh; margin: auto"
-    >
-      <GMapMarker
-          :key="index"
-          v-for="(marker, index) in markers"
-          :position="marker.position"
-          :clickable="true"
-          :draggable="true"
-          @click="openMarker(marker.id)"
+  <div class="content-wrapper">
+    <l-map v-bind="mapOptions">
+      <l-tile-layer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
```

Comment:
@Datikzz `{s}` and similar are being deprecated, see https://github.com/openstreetmap/operations/issues/737

Reply:
@danieldegroot2 thanks for hinting